src/parser/AdminSentences.h

```diff
@@ -248,6 +249,15 @@ class SpaceOptItem final {
         }
     }
 
+    std::string getGroupName() const {
+        if (isString()) {
+            return asString();
+        } else {
+            LOG(ERROR) << "group name value illage.";
+            return 0;
```

Comment:
Does this compile?

Reply:
it seems ok `getCharset` `getCollate` dose it with same way

Reply:
But it will crash.